docs/building-a-form/available-widgets.md

```diff
@@ -0,0 +1,100 @@
+# Available widgets
+
+These widgets are included in the us-forms-system by default. Set these widgets in a config file while building your form.
+
+Some widgets are associated with particular schema types or formats. There are additional widgets available, but the us-forms-system uses definitions they're included in, rather than just the field. To use a widget, set `ui:widget` for a field to the name of the widget. Widgets are located in [/src/js/widgets](../../src/js/widgets).
+
+Some widgets are passed as a string (e.g., `'ui: widget': 'date'`), while others are passed as a reference to the component itself (e.g., `'ui:widget': CurrencyWidget`).
+- Widgets from the [react-jsonschema-form library](./about-the-us-forms-system-library#understanding-react-jsonschema-form-rjsf) include string mappings.
+- Widgets created specifically for US Forms System do not have these mappings, and therefore must be specified by passing the component for the widget directly to the config.
```

Comment:
Now I'm starting to wonder about splitting up the widgets and definitions. For most of the `uiSchema`-only widgets I think users are most likely to include them via the `js/definitions` that  exist rather than using `'ui:widget'`: USFormsWidgetName`?

If that's the way they plan to do it, we need to explain here that they need to import the definitions in JavaScript near the top of their file:
```js
import CurrencyWidgetUI from 'us-forms-system/lib/js/definitions/currency';
```
Then use the widget by calling it to add all the `uiSchema` definitions:
```js
uiSchema: {
  ...
  monthlyWages: CurrencyWidgetUI('Monthly wages')
  ...
}
```
People *do* have the option of including the widget directly as defined here, but even then they would need to `import` the widget [like the definitions do](https://github.com/usds/us-forms-system/blob/master/src/js/definitions/currency.js). Note that the definitions (like this one) may do quite a few things so trying to explain manual usage when there are definitions may not be productive. 

@annekainicUSDS can you sanity-check what I'm saying on this? I think the **Usage:** instructions need to be modified to make them clear if this is the case.

Reply:
I've got some guidance here, could use some help with similar guidance for widgets: https://github.com/usds/us-forms-system/pull/182/commits/97e13c77ce19a99de5c9e60a86424331e18aa19e

Reply:
Added some comments on the above commit!

Reply:
For widgets, the similar text would be something like:
---
To include a widget, import the widget at the top of the file:
```js
import CurrencyWidget from 'us-forms-system/lib/js/widgets/CurrencyWidget';
```
Then and set the `ui:widget` field to the imported widget name:
```js
uiSchema: {
  ...
  'ui:widget': CurrencyWidget,
  ...
}
```

Reply:
Here's that information: https://github.com/usds/us-forms-system/pull/182/commits/bb892ab32d3c1e52941ecdf592543dd8665772bb